.github/workflows/check.yaml

```diff
@@ -1,36 +1,74 @@
 ---
-name: Check 🛠
+  name: Check 🛠
```

Comment:
Maybe the check workflow running in Docker could be placed in a separate yaml file?